src/main/java/com/amazonaws/cloudhsm/examples/AESWrappingRunner.java

```diff
@@ -71,6 +72,58 @@ public static void main(String[] args) throws Exception {
         aesWrappingKey.destroy();
         payloadKey.destroy();
     }
+    
+    /**
+     * This method demonstrates "AES/GCM/NoPadding"
+     * @param aesWrappingKey AES key for wrapping
+     * @param payloadKey Some payload-key to be wrapped
+     * @throws Exception
+     */
+    private static void aesGcmWrapNoPadding(Key aesWrappingKey, Key payloadKey) throws Exception {
+        // Create an AESWrap no padding cipher from the CloudHSM provider.
+        final Cipher cipher =
+                Cipher.getInstance("AES/GCM/NoPadding", CloudHsmProvider.PROVIDER_NAME);
+
+        //Initialization Vector - this is overridden by HSM
+        SecureRandom random = new SecureRandom();
+        byte[] IV = new byte[12];
+        random.nextBytes(IV);
+
+        // Create GCMParameterSpec GCMParameterSpec(int tagLen, byte[] iv)
+        // Can also use GCMParameterSpec(int tagLen, byte[] iv, int offset, int byteCount)
+        GCMParameterSpec gcmSpecWrap = new GCMParameterSpec(128, IV);
+
+        // Initialize the cipher in wrap mode.
+        cipher.init(Cipher.WRAP_MODE, aesWrappingKey, gcmSpecWrap);
+
+        // Wrap the payload key.
+        byte[] wrappedKey = cipher.wrap(payloadKey);
+        // Get iv from wrap cipher to perform unwrapping
+        byte[] iv = cipher.getIV();
+
+        // Initialize an AESWrap no padding cipher for unwrapping.
+        // Unwrap the wrapped key using the wrapping key.
+        GCMParameterSpec gcmSpecUnwrap = new GCMParameterSpec(128, iv);
```

Comment:
If we have `final int tagLen`, that would work nicely here.

```suggestion
        GCMParameterSpec gcmSpecUnwrap = new GCMParameterSpec(tagLen, iv);
```